app/src/main/java/me/saket/dank/ui/subreddit/uimodels/SubredditSubmission.java

```diff
@@ -197,7 +199,7 @@ public void renderPartialChanges(List<Object> payloads, SubmissionSwipeActionsPr
     }
 
     private void setThumbnail(Optional<UiModel.Thumbnail> optionalThumbnail) {
-      thumbnailView.setVisibility(uiModel.thumbnail().isPresent() ? View.VISIBLE : View.GONE);
+      //thumbnailView.setVisibility(uiModel.thumbnail().isPresent() ? View.VISIBLE : View.GONE);
```

Comment:
I don't think that this change is necessary. Your code should work without it. 
Right now it just breaks the setting which disables thumbnails.

Reply:
Fixed.